pkg/ui/workspaces/cluster-ui/src/jobs/jobDetailsPage/jobDetails.tsx

```diff
@@ -40,8 +42,13 @@ import {
 } from "../../api";
 import { Timestamp } from "../../timestamp";
 import { isTerminalState } from "../util/jobOptions";
+import { ColumnDescriptor, SortSetting, SortedTable } from "src/sortedtable";
+
+type JobMessage = cockroach.server.serverpb.IJobMessage;
```

Comment:
nit: We can probably replace this with:
```
type JobMessage = JobResponse['messages']
```
Will spare us the additional exported type in `jobsApi.ts`.

Reply:
I think the type of `messages` is []Message so that doesn't seem to work

Reply:
Oh whoops, right you would do `JobResponse['message'][number]`